lego/apps/users/serializers/users.py

```diff
@@ -60,6 +60,7 @@ class Meta:
             "full_name",
             "gender",
             "profile_picture",
+            "email_address",
```

Comment:
This will potentially return the private email of users without @abakus.no-email. Don't you intend to use `internal_email_address` instead?

Reply:
Didn't realize the user email was private, will change.